pkg/controller/servicebindingrequest/planner.go

```diff
@@ -76,7 +76,18 @@ var EmptyBackingServiceSelectorsErr = errors.New("backing service selectors are
 // Plan by retrieving the necessary resources related to binding a service backend.
 func (p *Planner) Plan() (*Plan, error) {
 	ns := p.sbr.GetNamespace()
-	selectors := append([]v1alpha1.BackingServiceSelector{}, p.sbr.Spec.BackingServiceSelectors...)
+
+	var selectors = []v1alpha1.BackingServiceSelector{}
+	if p.sbr.Spec.Services != nil {
+		selectors = append(selectors, *p.sbr.Spec.Services...)
```

Comment:
Removing the pointer in the `Services` definition removes the need for dereference here.